src/contexts/WalletContext/services.ts

```diff
@@ -219,25 +235,31 @@ export async function getWalletRiskAssessment(walletId: string) {
  */
 export const isWalletRisky = (riskAssessment?: WalletRiskAssessment) => {
   const indicators = riskAssessment?.addressRiskIndicators;
-  const redFlag = Array.isArray(indicators) && indicators.find(i => i.categoryRiskScoreLevel >= 10);
+  const redFlag =
+    Array.isArray(indicators) && indicators.find((i) => i.categoryRiskScoreLevel >= 10);
```

Comment:
```suggestion
    Array.isArray(indicators) && indicators.some((i) => i.categoryRiskScoreLevel >= 10);
```

perhaps more performant since it returns bool and not the item (object)

Reply:
Open to this change but I don't think this PR is the right place cos I haven't touched this function (other than it being auto-formatted by prettier).

Reply:
Alrighty ^_^ I just saw it as opportunity for improvement. It won't break anything